.pre-commit-config.yaml

```diff
@@ -0,0 +1,28 @@
+repos:
+  - repo: https://github.com/adamchainz/blacken-docs
+    rev: "1.16.0"
+    hooks:
+      - id: blacken-docs
+        additional_dependencies:
+        - black==24.2.0
+
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.5.0
+    hooks:
```

Comment:
Do we currently pass all the hooks with main?

Reply:
Nope, but I will soon open a PR to fix this.